CombinatorialGames/Surreal/Basic.lean

```diff
@@ -233,10 +239,10 @@ instance : PartialOrder Surreal :=
   inferInstanceAs (PartialOrder (Antisymmetrization ..))
 
 instance : LinearOrderedAddCommGroup Surreal where
-  zero_add := by rintro ⟨x⟩; change mk (0 + _) = mk _; simp_rw [zero_add]
-  add_zero := by rintro ⟨x⟩; change mk (_ + 0) = mk _; simp_rw [add_zero]
-  add_comm := by rintro ⟨x⟩ ⟨y⟩; change mk (_ + _) = mk (_ + _); simp_rw [add_comm]
-  add_assoc := by rintro ⟨x⟩ ⟨y⟩ ⟨z⟩; change mk (_ + _ + _) = mk (_ + (_ + _)); simp_rw [add_assoc]
+  zero_add := by rintro ⟨x⟩; change mk (0 + x) = mk x; simp_rw [zero_add]
```

Comment:
I do prefer the explicit variable usage since `rintro`'s destructured introduced variable name is explicit.